contributing/documentation/overview.rst

```diff
@@ -244,10 +244,25 @@ steps to contribute to the Symfony documentation, which you can use as a
 
     # (optional) make the changes requested by reviewers and commit them
     $ git commit xxx.rst
-    $ git push
+    $ git push origin my_changes
 
 You guessed right: after all this hard work, it's **time to celebrate again!**
 
+Minor Changes (e.g. Typos)
+--------------------------
+
+You may find just a typo and want to fix it. Due to GitHub's functional
+frontend, it is quite simple to create Pull Requests right in your
+browser while reading the docs on **symfony.com**. To do this, just click
+the ``edit this page`` button on the upper right corner. Beforehand
```

Comment:
I agree. I used this to be consistent with the existing 
> click on the big `New pull request` button

in step 7. I will make both bold.